lib/cocina/models/access.rb

```diff
@@ -4,7 +4,7 @@ module Cocina
   module Models
     class Access < Struct
       # Access level
-      attribute :access, Types::Strict::String.default('dark').enum('world', 'stanford', 'location-based', 'citation-only', 'dark').meta(omittable: true)
+      attribute :access, Types::Strict::String.default('dark').enum('world', 'stanford', 'location-based', 'citation-only', 'dark', 'cdl').meta(omittable: true)
```

Comment:
This should be a separate property as we do for DRO access

Reply:
We make changes to this project by updating openapi.yml and running the generator